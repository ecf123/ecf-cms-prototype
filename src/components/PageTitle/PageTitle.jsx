import React from "react";
import "./PageTitle.scss";

const PageTitle = ({ title, className }) => {
  return <h1 className={className}>{title}</h1>;
};

export default PageTitle;
