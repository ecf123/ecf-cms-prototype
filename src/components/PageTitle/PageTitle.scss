@use "../../styles/variables" as *;

.page-title {
  font-family: $font-secondary;
  font-weight: $font-weight-regular;
  color: $color-blue-navy-dark;

  &--small {
    font-size: $font-size-extra-large;
  }

  &--large {
    font-size: 4rem;
  }
}
