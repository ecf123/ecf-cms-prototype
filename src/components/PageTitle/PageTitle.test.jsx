import { render, screen} from "@testing-library/react";
import { customRender } from "../../utils/testUtils";
import PageTitle from "./PageTitle";

it("should have a title displayed", () => {
    render(<PageTitle title="test title"/>);
    const title = screen.getByText("test title");
    expect(title).toBeInTheDocument();
})

it("Should render layout with children", () => {
  const { container } = customRender(
    <PageTitle title="test title"/>
  );
  expect(container).toMatchSnapshot();
});