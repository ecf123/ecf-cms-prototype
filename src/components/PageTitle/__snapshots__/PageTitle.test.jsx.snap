// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render layout with children 1`] = `
<div>
  <h1>
    test title
  </h1>
</div>
`;
